nexus-config/src/nexus_config.rs

```diff
@@ -856,6 +857,12 @@ pub struct PackageConfig {
     /// Maghemite mgd daemon configuration
     #[serde(default)]
     pub mgd: HashMap<SwitchLocation, MgdConfig>,
+    /// Initial reconfigurator chicken switches
+    ///
+    /// We use this hook to disable reconfigurator automation in the test suite
+    #[serde(default)]
+    pub initial_reconfigurator_chicken_switches:
```

Comment:
Discussed this a little in chat, but wanted to make it public:

The usage of this `planner_enabled` option for tests is 100% reasonable, but not at all a "chicken switch":
- We aren't leaving it as a toggle-able option to be conservative, and for enabling later
- ... it's "just a config option", which we want to have set in some circumstances, and unset in others.

It probably behooves us to rename `ReconfiguratorChickenSwitch` to `ReconfiguratorConfig`, if the list of settings is going to be long-lasting (which is fine! It's just confusing to call it a "chicken switch").

Reply:
Follow-up: Obviously, doesn't need to block this PR.